chall02/agarzon-.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+
+morse = {

```

Comment:
If you use globals in Python, write them in upper case.
Here in your dictionary you could write the letter in upper or lower case, but not both and then change all what you receive in upper or lower case in order to retreive the character in the dictionary.